topics/metagenomics/tutorials/general-tutorial/tutorial.md

```diff
@@ -385,7 +385,7 @@ Have a look at the taxonomy output. You will see that every read now has a class
 
 The next step is then to use this information to know the abundance of the different found taxons.
 
-> ### {% icon hands_on %} Hands-on: Cluster our data into OTUs
+> ### {% icon hands_on %} Hands-on: Cluster our data into OTUs TODO how is the taxonomy used here? Are actually only the phylotypes clustered into OTUs?
```

Comment:
did you find the answer to this TODO? (could you put it as a comment (`<!-- -->`) if not?)

Reply:
(sorry, didnt see your comment about the TODOs in description) I will see if I can find the answer to this